<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AiPromptDialog</class>
 <widget class="QDialog" name="AiPromptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI创作</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="modelComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DeepSeek-V3-0324</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeepSeek-R1-0528</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OpenAI (GPT-3.5)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OpenAI (GPT-4 Turbo)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入你的绘图指令：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="promptTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>API密钥 (Key)：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="apiKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AiPromptDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AiPromptDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
